dask_ml/model_selection/_hyperband.py

```diff
@@ -341,9 +349,10 @@ def _get_SHAs(self, brackets):
                 test_size=self.test_size,
                 random_state=seed_start + b if b != 0 else self.random_state,
                 scoring=self.scoring,
+                verbose=self.verbose,
             )
-            for b, (n, r) in brackets.items()
-        }
+            sha._prefix = ", bracket=" + str(b)
```

Comment:
FWIW, I think it's fine to add `prefix` as a parameter to SuccessiveHalvingSearchCV, rather than setting it after the fact here. That makes it a bit more robust to thinks like `clone`.

Reply:
Now, only `SuccessiveHalvingSearchCV` has  a `prefix` argument. `IncrementalSearchCV` is un-modified (which made for a slightly unorthodox implementation).